DrillSergeant/neos-development-collection
# Hide property "target" of nodeType "Neos.Neos:Shortcut" when targetMode not matching

**What I did**/**How I did it**
I added configuration to the shortcut nodetype definition to hide the target-property in the UI when targetMode is not "selectedTarget"

**How to verify it**
Insert a shortcut node. You should *not* see the target-property untill you change the targetMode to "selectedTarget"
